client/src/components/FullLogo/Logo.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "./Logo.css";
+
+const Logo = props => {
+  let classList = "logo-tag";
+
+  let types = ["logo", "no-tag"];
+
+  if (types.includes(props.type)) {
+    classList += ` ${props.type}`;
+  }
+
+  return (
+    <>
+      <div className="logo-tag">
+        <div className="logo">
+          <div className="modern">Modern</div>
+          <div className="trouseau">trousseau</div>
```

Comment:
Not sure if the misspelling was intentional but the company name is spelled with two `s` not one. 
It should be `trousseau` not `trouseau`.